### PR TITLE
Add Efficient Interpretation using Quickening paper.

### PR DESCRIPTION
**Pull request by Pavel-Durov**

### CODE REVIEW

File: softdev.bib [hunk 1]
```diff
@@ -3342,3 +3342,11 @@ @inproceedings{cheng20typefreezing
     month = Feb,
     booktitle = "CGO"
 }
+
+@inproceedings{brunthaler10quickening,
+    title = "Efficient Interpretation using Quickening",
+    author = "Brunthaler, Stefan",
+    year = 2010,
+    month = Oct,
+    booktitle = {DLS}
```
Comment (ltratt) [inline softdev.bib:3351]: Is there a DOI for this? If so, it's good to add it, as some LaTeX styles make the links clickable.

Comment (Pavel-Durov) [reply softdev.bib:3352]: Added :point_right: b8b60873b93b5b40cbd181a4edb8c26d25a6f60c

**Comment by ltratt** [comment]
Please squash.

**Comment by Pavel-Durov** [comment]
Squashed :point_right: 68da56a7f62818bf63d1594564ad63089ec1bed6

**Pull request merged**